### PR TITLE
Adds the audit report to our repo

### LINKED ISSUE #1339: Add the security audit report to the repo
Following a post in the forum, we'd like to add the final report to the repo for easy reference.

The tasks involve:

- Finding a good place to put it, Should it be in the root?
- Updating the readme to have a note that links to the report.

The fie is 427kb, would that be a problem to add it to the repo?

[Least_Authority_-_Blockstack_-_Stacks_Wallet_Extension_-_Final_Audit_Report.pdf](https://github.com/blockstack/stacks-wallet-web/files/6702223/Least_Authority_-_Blockstack_-_Stacks_Wallet_Extension_-_Final_Audit_Report.pdf)

### PR DESCRIPTION
**Pull request by andresgalante**
> Try out this version of the Stacks Wallet - download [extension builds](https://github.com/blockstack/ux/actions/runs/987362486).

Fixes #1339 by adding the security audit report to our repo

### CODE REVIEW

**Comment by kyranjamie** [comment]
Perhaps the package can be named something nice than `04292021-Least_Authority_-_Hiro_-_Stacks_Wallet_Extension_-_Final_Audit_Report.pdf`

`least-authority-security-audit-report.pdf`?

**Comment by kyranjamie** [comment]
@andresgalante builds are failing as you're not using our commit convention: 

should be `chore: add security report to repo, closes #1339`

**Comment by andresgalante** [comment]
Hopefully I didn't screw things up too much this time :)

File: README.md [hunk 1]
```diff
@@ -137,3 +139,21 @@ Alternatively, for Firefox:
...
+If you discover a security vulnerability, please use one of the following means of communications to report it to us:
+
+- Report the security issue to our [HackerOne program](https://hackerone.com/hiro)
+- Report the security issue directly at [security@hiro.so](mailto:security@hiro.so)
+
+Please note this email is strictly for reporting security vulnerabilities. For support queries, contact wallet@hiro.so. Your efforts to responsibly disclose your findings are sincerely appreciated and will be taken into account to acknowledge your contributions.
+
+### Audit Report
+
+In Q1 2021, Hiro partnered with Least Authority, a leading security consultancy with experience in the crypto space, to audit Hiro's Stacks Wallet for Web. On April 29th 2021, after addressing the major concerns described in the initial findings, as well as a concluding sign off from the Least Authority team, a final report was delivered.
```
Comment (aulneau) [inline README.md]:
```suggestion
In Q1 2021, Hiro partnered with [Least Authority](https://leastauthority.com/), a leading security consultancy with experience in the crypto space, to audit Hiro's Stacks Wallet for Web. On April 29th 2021, after addressing the major concerns described in the initial findings, as well as a concluding sign off from the Least Authority team, a final report was delivered.
```

**Review by aulneau — APPROVED** [review]

**Pull request merged**